wandb/sdk/wandb_run.py

```diff
@@ -2705,27 +2705,40 @@ def define_metric(
         overwrite: Optional[bool] = None,
         **kwargs: Any,
     ) -> wandb_metric.Metric:
-        """Define metric properties which will later be logged with `wandb.log()`.
+        """Customize metrics logged with `wandb.log()`.
 
         Arguments:
-            name: Name of the metric.
-            step_metric: Independent variable associated with the metric.
-            step_sync: Automatically add `step_metric` to history if needed.
-                Defaults to True if step_metric is specified.
+            name: The name of the metric to customize.
+            step_metric: The name of another metric to serve as the X-axis
+                for this metric in automatically generated charts.
+            step_sync: Automatically insert the last value of step_metric into
+                `run.log()` if it is not provided explicitly. Defaults to True
+                 if step_metric is specified.
             hidden: Hide this metric from automatic plots.
             summary: Specify aggregate metrics added to summary.
-                Supported aggregations: "min,max,mean,best,last,none"
-                Default aggregation is `copy`
-                Aggregation `best` defaults to `goal`==`minimize`
-            goal: Specify direction for optimizing the metric.
-                Supported directions: "minimize,maximize"
+                Supported aggregations include "min", "max", "mean", "last",
+                "best", "copy" and "none". "best" is used together with the
+                goal parameter. "none" prevents a summary from being generated.
```

Comment:
from my understanding `copy` is mutually exclusive with all the other ones, as it just does the default `summary` same as if we didn't specify anything. basically, this code:
```
run.define_metric("metric", summary="min,copy")

for i in range(10):
    run.log({"metric": i})
```
(in non `wandb-core`) will return `summary={"metric": 9}`

Reply:
should we just remove the mention of `copy` it is basically the same as not specifying anything?

Reply:
```
run.define_metric("metric")

for i in range(10):
    run.log({"metric": i})
```

Reply:
or add deprecation message for people that used it?

Reply:
let's add a deprecation warning and remove it in 0.18

Reply:
added